src/sbt-test/sbt-scalafix/root-validation/build.sbt

```diff
@@ -0,0 +1,20 @@
+inThisBuild(
+  List(
+    scalaVersion := "2.12.6",
+  )
+)
+
+lazy val root = project
```

Comment:
@abdheshkumar I was able to reproduce the bug with this build.sbt, I got the following error before the fix

```
[info] [error] (Compile / scalafix) scalafix.sbt.InvalidArgument: 2 errors
[info] [error] [E1] The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[info] [error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[info] [error] To fix this problem permanently for your build, add the following settings to build.sbt:
[info] [error]   addCompilerPlugin(scalafixSemanticdb)
[info] [error]   scalacOptions += "-Yrangepos"
[info] [error]
[info] [error] [E2] The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
[info] [error] To fix this problem, update your build to use at least one Scala compiler
[info] [error] option that starts with "-Ywarn-unused"
```

After the fix `scalafix` runs successfully.